src/formatDate.js

```diff
@@ -50,7 +50,43 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  let day = '';
+  let month = '';
+  let year = '';
+  let result = '';
+  const splits = date.split(fromFormat[3]);
+
```

Comment:
Let's assign this index of a "/" to a variable for it to be meaningful. And maybe use another variable instead of splits, maybe "dateParts" or "dateArray".

Reply:
That I can change.

Reply:
okay